include/aspect/parameters.h

```diff
@@ -495,6 +495,7 @@ namespace aspect
      */
     bool                           include_melt_transport;
     bool                           enable_additional_stokes_rhs;
+    bool                           enable_prescribed_compression;
```

Comment:
why have a flag? can't you just test whether the material model attached the appropriate additional output?